.github/workflows/cluster_cleanup.yml

```diff
@@ -71,8 +75,13 @@ jobs:
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
       - name: Delete keyvaults
         run: |
-          for vault in $(az keyvault list --query "[?starts_with(name, '${{ env.KEYVAULT_PREFIX }}')].name" -o tsv); do
-            az keyvault delete -n $vault -g ${{ env.RESOURCE_GROUP }} --no-wait
+          for vault in $(az keyvault list --query "[?starts_with(name, '${{ inputs.keyvault_prefix }}')].name" -o tsv); do
```

Comment:
This wasn't in the previous command for some reason, but I'm thinking we should probably limit the list results to our particular resource group

Reply:
list has an easy -g but list-deleted does not D:

Dammit will need to make more jmes queries